recipes-qt/qt5/qtbase_git.bb

```diff
@@ -31,6 +31,7 @@ SRC_URI += "\
     file://0013-Upgrade-double-conversion-to-v3.0.0.patch \
     file://0014-Check-glibc-version-for-renameat2-statx-on-non-boots.patch \
     file://0015-double-conversion-support-AARCH64EB-and-arm-BE.patch \
+    file://0001-Qt5GuiConfigExtras.cmake.in-cope-with-variable-path-.patch \
```

Comment:
The comment above SRC_URI says:
# common for qtbase-native, qtbase-nativesdk and qtbase
yet you've added it only for target qtbase.

Reply:
Ok, I see.

For me 'common for qtbase-native, qtbase-nativesdk and qtbase' sounds like this is picked up by all 3 builds, not a hint that 3 recipes must be kept in sync.

Will send a follow-up pull request after building and testing the changes.

Reply:
Pushed a changed commit which added the patch to the SRC_URI in qtbase-native and nativesdk-qtbase.